code/Collector/ConfigCollector.php

```diff
@@ -53,8 +54,8 @@ public function getAssets()
         $name = $this->getName();
 
         return [
-            'base_path' => '/' . DEBUGBAR_DIR . '/javascript',
-            'base_url' => DEBUGBAR_DIR . '/javascript',
+            'base_path' => '/' . Director::makeRelative(ModuleLoader::getModule('lekoala/silverstripe-debugbar')->getResource('javascript')->getURL()),
```

Comment:
Please use `$resource->getRelativePath()` instead of `Director::makeRelative($resource->getURL())`

Reply:
I was having enormous trouble getting that to work as my dev environment lives on a sub-url, i.e. 
`http://dev/stabless4` - and I could never get the right path. This is the only one that worked. I will  try again tomorrow but I want `getRelativeURL()` and there wasn't an option :)

Reply:
Yes, but url isn't a path... you could have a resource that lives locally, but is served via a CDN, in which case there won't be any relation between the url and the path at all.